### PR TITLE
🌐 Add Portuguese translation for `docs/pt/docs/tutorial/schema-extra-example.md`

### PR DESCRIPTION
**Pull request by luccasmmg**

### CODE REVIEW

File: docs/pt/docs/tutorial/schema-extra-example.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+Você pode declarar um `example` para um modelo Pydantic usando `Config` e `schema_extra`, conforme descrito em <a href="https://pydantic-docs.helpmanual.io/usage/schema/#schema-customization" class="external-link" target="_blank">Documentação do Pydantic: Schema customization</a>:
+
+```Python hl_lines="15-23"
+{!../../../docs_src/schema_extra_example/tutorial001.py!}
+```
+
+Essas informações extras serão adicionadas como se encontram no **JSON Schema** de resposta desse modelo e serão usadas na documentação da API.
+
+!!! dica
```
Comment (lsglucas) [inline docs/pt/docs/tutorial/schema-extra-example.md:17]:
```suggestion
!!! tip "dica"
```

File: docs/pt/docs/tutorial/schema-extra-example.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+### Exemplos na UI da documentação
+
+Com `examples` adicionado a `Body()`, os `/docs` vão ficar assim:
+
+<img src="/img/tutorial/body-fields/image02.png">
+
+## Detalhes técnicos
+
+!!! atenção
```
Comment (lsglucas) [inline docs/pt/docs/tutorial/schema-extra-example.md:88]:
```suggestion
!!! warning "atenção"
```

**Review by stlucasgarcia — CHANGES_REQUESTED** [review]

File: docs/pt/docs/tutorial/schema-extra-example.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+Portanto, embora `example` não seja parte do JSON Schema, é parte da versão customizada do JSON Schema usada pelo OpenAPI, e é isso que vai ser usado dentro da UI de documentação.
+
+Mas quando você usa `example` ou `examples` com qualquer um dos outros utilitários (`Query ()`, `Body ()`, etc.) esses exemplos não são adicionados ao JSON Schema que descreve esses dados (nem mesmo para versão própria do OpenAPI do JSON Schema), eles são adicionados diretamente à declaração da *operação de rota* no OpenAPI (fora das partes do OpenAPI que usam o JSON Schema).
+
+Para `Path()`, `Query()`, `Header()`, e `Cookie()`, o `example` e `examples` são adicionados a<a href="https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.0.3.md#parameter-object" class="external-link" target="_blank">definição do OpenAPI, dentro do `Parameter Object` (na especificação)</a>.
```
Comment (Lahiry) [inline docs/pt/docs/tutorial/schema-extra-example.md:105]:
```suggestion
Para `Path()`, `Query()`, `Header()`, e `Cookie()`, o `example` e `examples` são adicionados a <a href="https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.0.3.md#parameter-object" class="external-link" target="_blank">definição do OpenAPI, dentro do `Parameter Object` (na especificação)</a>.
```

File: docs/pt/docs/tutorial/schema-extra-example.md [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+
+E para `Body()`, `File()`, e `Form()`, o `example` e `examples` são de maneira equivalente adicionados para a <a href="https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.0.3.md#mediaTypeObject" class="external-link" target="_blank">definição do OpenAPI, dentro do `Request Body Object`, no campo `content`, no `Media Type Object` (na especificação)</a>.
+
+Por outro lado, há uma versão mais recente do OpenAPI: **3.1.0**, lançada recentemente. Baseado no JSON Schema mais recente e a maioria das modificações da versão customizada do OpenAPI do JSON Schema são removidas, em troca dos recursos das versões recentes do JSON Schema, portanto, todas essas pequenas diferenças são reduzidas. No entanto, a UI do Swagger atualmente não oferece suporte a OpenAPI 3.1.0, então, por enquanto, é melhor continuar usando as ideias acima.
```
Comment (Lahiry) [inline docs/pt/docs/tutorial/schema-extra-example.md:109]:
```suggestion
Por outro lado, há uma versão mais recente do OpenAPI: **3.1.0**, lançada recentemente. Baseado no JSON Schema mais recente e a maioria das modificações da versão customizada do OpenAPI do JSON Schema são removidas, em troca dos recursos das versões recentes do JSON Schema, portanto, todas essas pequenas diferenças são reduzidas. No entanto, a UI do Swagger atualmente não oferece suporte a OpenAPI 3.1.0, então, por enquanto, é melhor continuar usando as opções acima.
```

**Review by Lahiry — CHANGES_REQUESTED** [review]
Just need to fix some spacing and a word choice, good job!

**Review by stlucasgarcia — APPROVED** [review]

File: docs/pt/docs/tutorial/schema-extra-example.md [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+```Python hl_lines="15-23"
+{!../../../docs_src/schema_extra_example/tutorial001.py!}
+```
+
+Essas informações extras serão adicionadas como se encontram no **JSON Schema** de resposta desse modelo e serão usadas na documentação da API.
+
+!!! tip "dica"
+    Você pode usar a mesma técnica para estender o JSON Schema e adicionar suas próprias informações extras de forma personalizada.
+
+    Por exemplo, você pode usar isso para adicionar metadados em uma interface de usuário de front-end, etc.
```
Comment (LorhanSohaky) [inline docs/pt/docs/tutorial/schema-extra-example.md:20]:
```suggestion
    Por exemplo, você pode usar isso para adicionar metadados para uma interface de usuário de front-end, etc.
```

File: docs/pt/docs/tutorial/schema-extra-example.md [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+
+Você pode declarar um `example` para um modelo Pydantic usando `Config` e `schema_extra`, conforme descrito em <a href="https://pydantic-docs.helpmanual.io/usage/schema/#schema-customization" class="external-link" target="_blank">Documentação do Pydantic: Schema customization</a>:
+
+```Python hl_lines="15-23"
+{!../../../docs_src/schema_extra_example/tutorial001.py!}
+```
+
+Essas informações extras serão adicionadas como se encontram no **JSON Schema** de resposta desse modelo e serão usadas na documentação da API.
+
+!!! tip "dica"
```
Comment (LorhanSohaky) [inline docs/pt/docs/tutorial/schema-extra-example.md:17]:
```suggestion
!!! tip "Dica"
```

File: docs/pt/docs/tutorial/schema-extra-example.md [hunk 7]
```diff
@@ -0,0 +1,109 @@
...
+
+Ao usar `Field ()` com modelos Pydantic, você também pode declarar informações extras para o **JSON Schema** passando quaisquer outros argumentos arbitrários para a função.
+
+Você pode usar isso para adicionar um `example` para cada campo:
+
+```Python hl_lines="4  10-13"
+{!../../../docs_src/schema_extra_example/tutorial002.py!}
+```
+
+!!! warning "atenção"
```
Comment (LorhanSohaky) [inline docs/pt/docs/tutorial/schema-extra-example.md:32]:
```suggestion
!!! warning "Atenção"
```

File: docs/pt/docs/tutorial/schema-extra-example.md [hunk 8]
```diff
@@ -0,0 +1,109 @@
...
+
+* `Path()`
+* `Query()`
+* `Header()`
+* `Cookie()`
+* `Body()`
+* `Form()`
+* `File()`
+
+você também pode declarar um dado `example` ou um grupo de `examples` com informações adicionais que serão adicionadas à **OpenAPI**.
```
Comment (LorhanSohaky) [inline docs/pt/docs/tutorial/schema-extra-example.md:47]:
```suggestion
você também pode declarar um dado `example` ou um grupo de `examples` com informações adicionais que serão adicionadas ao **OpenAPI**.
```

File: docs/pt/docs/tutorial/schema-extra-example.md [hunk 9]
```diff
@@ -0,0 +1,109 @@
...
+
+Aqui nós passamos um `example` dos dados esperados por `Body()`:
+
+```Python hl_lines="21-26"
+{!../../../docs_src/schema_extra_example/tutorial003.py!}
+```
+
+### Exemplo na UI da documentação
+
+Com qualquer um dos métodos acima, os `/docs` vão ficar assim:
```
Comment (LorhanSohaky) [inline docs/pt/docs/tutorial/schema-extra-example.md:59]:
```suggestion
Com qualquer um dos métodos acima, ficaria assim na `/docs`:
```

File: docs/pt/docs/tutorial/schema-extra-example.md [hunk 10]
```diff
@@ -0,0 +1,109 @@
...
+
+<img src="/img/tutorial/body-fields/image01.png">
+
+### `Body` com vários `examples`
+
+Alternativamente ao único `example`, você pode passar `examples` usando um `dict` com **vários examples**, cada um com informações extras que serão adicionadas no **OpenAPI** também.
```
Comment (LorhanSohaky) [inline docs/pt/docs/tutorial/schema-extra-example.md:65]:
```suggestion
Alternativamente ao único `example`, você pode passar `examples` usando um `dict` com **vários exemplos**, cada um com informações extras que serão adicionadas no **OpenAPI** também.
```

File: docs/pt/docs/tutorial/schema-extra-example.md [hunk 11]
```diff
@@ -0,0 +1,109 @@
...
+
+As chaves do `dict` identificam cada exemplo, e cada valor é outro `dict`.
+
+Cada `dict` de exemplo específico em `examples` pode conter:
+
+* `summary`: Pequena descrição para o exemplo.
```
Comment (LorhanSohaky) [inline docs/pt/docs/tutorial/schema-extra-example.md:71]:
```suggestion
* `summary`: Pequena descrição do exemplo.
```

File: docs/pt/docs/tutorial/schema-extra-example.md [hunk 12]
```diff
@@ -0,0 +1,109 @@
...
+* `description`: Uma descrição longa que pode conter texto em Markdown.
+* `value`: O próprio exemplo mostrado, ex: um `dict`.
+* `externalValue`: alternativo a `value`, uma URL apontando para o exemplo. Embora isso possa não ser suportado por tantas ferramentas quanto `value`.
```
Comment (LorhanSohaky) [inline docs/pt/docs/tutorial/schema-extra-example.md:74]:
```suggestion
* `externalValue`: alternativa ao `value`, uma URL apontando para o exemplo. Embora isso possa não ser suportado por tantas ferramentas quanto `value`.
```

File: docs/pt/docs/tutorial/schema-extra-example.md [hunk 13]
```diff
@@ -0,0 +1,109 @@
...
+
+```Python hl_lines="22-48"
+{!../../../docs_src/schema_extra_example/tutorial004.py!}
+```
+
+### Exemplos na UI da documentação
+
+Com `examples` adicionado a `Body()`, os `/docs` vão ficar assim:
```
Comment (LorhanSohaky) [inline docs/pt/docs/tutorial/schema-extra-example.md:82]:
```suggestion
Com `examples` adicionados ao `Body()`, o `/docs` ficaria assim:
```

File: docs/pt/docs/tutorial/schema-extra-example.md [hunk 14]
```diff
@@ -0,0 +1,109 @@
...
+
+<img src="/img/tutorial/body-fields/image02.png">
+
+## Detalhes técnicos
+
+!!! warning "atenção"
```
Comment (LorhanSohaky) [inline docs/pt/docs/tutorial/schema-extra-example.md:88]:
```suggestion
!!! warning "Atenção"
```

File: docs/pt/docs/tutorial/schema-extra-example.md [hunk 15]
```diff
@@ -0,0 +1,109 @@
...
+    Esses são detalhes muito técnicos sobre os padrões **JSON Schema** e **OpenAPI**.
+
+    Se as ideias explicadas acima já funcionam para você, isso pode ser o suficiente, e você provavelmente não precisa desses detalhes, fique à vontade para pular.
+
+Quando você adiciona um exemplo dentro de um modelo Pydantic, usando `schema_extra` ou` Field (example = "something") `esse exemplo é adicionado ao **JSON Schema** para esse modelo Pydantic.
```
Comment (LorhanSohaky) [inline docs/pt/docs/tutorial/schema-extra-example.md:93]:
```suggestion
Quando você adiciona um exemplo dentro de um modelo Pydantic, usando `schema_extra` ou` Field(example="something") `esse exemplo é adicionado ao **JSON Schema** para esse modelo Pydantic.
```

**Review by LorhanSohaky — COMMENTED** [review]
I think with these modifications the reading will be more fluent

File: docs/pt/docs/tutorial/schema-extra-example.md (hunk 13 above)
Comment (LorhanSohaky) [reply docs/pt/docs/tutorial/schema-extra-example.md:82]: Missed that

File: docs/pt/docs/tutorial/schema-extra-example.md (hunk 9 above)
Comment (LorhanSohaky) [reply docs/pt/docs/tutorial/schema-extra-example.md:59]: Missed that

**Comment by tiangolo** [comment]
📝 Docs preview for commit 3bfd1df971fb45887313535fd168d81bb01603c8 at: https://649a1e53d1bbbc159e60f87f--fastapi.netlify.app

File: docs/pt/docs/tutorial/schema-extra-example.md [hunk 16]
```diff
@@ -0,0 +1,109 @@
...
+
+Por isso, o OpenAPI 3.0.3 definiu o seu próprio <a href="https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.3.md#fixed-fields-20" class="external-link" target="_blank">`example`</a> para a versão modificada do **JSON Schema** que é usada, para o mesmo próposito (mas é apenas `example` no singular, não `examples`), e é isso que é usado pela UI da documentação da API(usando o Swagger UI).
+
+Portanto, embora `example` não seja parte do JSON Schema, é parte da versão customizada do JSON Schema usada pelo OpenAPI, e é isso que vai ser usado dentro da UI de documentação.
+
+Mas quando você usa `example` ou `examples` com qualquer um dos outros utilitários (`Query ()`, `Body ()`, etc.) esses exemplos não são adicionados ao JSON Schema que descreve esses dados (nem mesmo para versão própria do OpenAPI do JSON Schema), eles são adicionados diretamente à declaração da *operação de rota* no OpenAPI (fora das partes do OpenAPI que usam o JSON Schema).
```
Comment (jessicapaz) [inline docs/pt/docs/tutorial/schema-extra-example.md:103]:
```suggestion
Mas quando você usa `example` ou `examples` com qualquer um dos outros utilitários (`Query()`, `Body()`, etc.) esses exemplos não são adicionados ao JSON Schema que descreve esses dados (nem mesmo para versão própria do OpenAPI do JSON Schema), eles são adicionados diretamente à declaração da *operação de rota* no OpenAPI (fora das partes do OpenAPI que usam o JSON Schema).
```

**Comment by tiangolo** [comment]
📝 Docs preview for commit b9e3edaf96b2939365035d0377e59f141762b3f0 at: https://64a3fb27ac978b0872bdbd79--fastapi.netlify.app

**Comment by tiangolo** [comment]
Hello @LorhanSohaky, @jessicapaz, @Lahiry! 👋 

Would you think this is ready after the changes? Would you approve it? 🤓

**Comment by tiangolo** [comment]
📝 Docs preview for commit eae3b01148eb496795e4e975c42ed6b89eb32910 at: https://0c8a6145.fastapitiangolo.pages.dev

**Comment by alejsdev** [comment]
Thanks for your contribution @luccasmmg 😎 

And thanks to everyone for the reviews 🥳🎉

**Pull request merged**